### ISSUE TITLE
Vocabulary problem with Text Data Type

### ISSUE THREAD

**Issue by alperkesen** [alperkesen/AutoML.jl#9]
Vocabulary should be created for only training set and with all texts in training set. For example, if there are two reviews, vocabulary should be initialized using these two texts. In the test set, the created vocabulary should be used to transform the words into ids.
